### PR TITLE
Home task HTML

### PR DESCRIPTION
**Pull request by arester1980**

### CODE REVIEW

File: avodolazov/01_html/task2.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<p>Введите эти 4 строки:</p>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+<p>Это всё. Сохраните файл как hello.py в любом месте. Я предлагаю сохранить его в папке
+python_examples в домашнем каталоге. Этот пример показывает, как просто написать программу на
+Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале <pre>
+chmod +x hello.py </pre>
+<p>в папке, в которой вы сохранили программу. Теперь запустите её.<p>
```
Comment (pluhin) [inline avodolazov/01_html/task2.html]: twice opened tag

File: avodolazov/01_html/task2.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<h2>Наша первая программа</h2>
+<p>Наберём код в текстовом редакторе (например, в gedit). Потом мы разберёмся, что делает каждая строка и пойдём дальше.</p>
+<p>Введите эти 4 строки:</p>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+<p>Это всё. Сохраните файл как hello.py в любом месте. Я предлагаю сохранить его в папке
+python_examples в домашнем каталоге. Этот пример показывает, как просто написать программу на
+Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале <pre>
```
Comment (pluhin) [inline avodolazov/01_html/task2.html]: Here need to cloese paragraph

File: avodolazov/01_html/task2.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+<h2>Пример цикла For</h2>
+<p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и
+наберите следующую программу:</p>
+
+<pre>#!/usr/bin/env python
+for cntr in range(0,10):
+print cntr</pre>
+<p>Не забудьте поставить символ табуляции перед строкой «print cntr». Это важно. В Python для
+указания на блоки кода используются не круглые «(» и фигурные «{» скобки как в других языках
+программирования, а отступы.</p>
```
Comment (pluhin) [inline avodolazov/01_html/task2.html]: Where is the end of the file? body and html did not close

File: avodolazov/01_html/task3.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+greg@earth:~/python_examples$</pre>
+<p>Вот и всё.</p>
+<h2>Пример цикла For</h2>
+<p>Теперь давайте рассмотрим пример программирования цикла. Запустите текстовый редактор и
+наберите следующую программу:</p>
+
+<pre>#!/usr/bin/env python
+for cntr in range(0,10):
+print cntr</pre>
+<p><em>Не забудьте поставить символ табуляции перед строкой «print cntr»</em>. Это важно. В Python для указания на блоки кода используются не круглые «(» и фигурные «{» скобки как в других языках программирования, а отступы.</p>
```
Comment (pluhin) [inline avodolazov/01_html/task3.html]: Where is the end of the file? body and html did not close

File: avodolazov/01_html/task3.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+<h2>Наша первая программа</h2>
+<p>Наберём код в текстовом редакторе (например, в <strong>gedit</strong>). Потом мы разберёмся, что делает каждая строка и пойдём дальше.</p>
+<p>Введите эти 4 строки:</p>
+<pre>#!/usr/bin/env python
+print 'Hello. I am a python program.'
+name = raw_input("What is your name?")
+print "Hello there, " + name + "!"</pre>
+<p>Это всё. Сохраните файл как <b>hello.py</b> в любом месте. Я предлагаю сохранить его в папке
+<b>python_examples</b> в домашнем каталоге. Этот пример показывает, как просто написать программу на
+Python. Для запуска программы нужно сделать её файл исполняемым. Для этого введите в терминале <pre><b><u><i>
```
Comment (pluhin) [inline avodolazov/01_html/task3.html]: Need to close p

File: avodolazov/01_html/task4.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам.
+Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят
+вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ,
+комментарии станут важными. Также при их помощи вы заставите Python игнорировать
+какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'.
+Например:</p>
+<pre>#</pre>
+<p>Это комментарий</p>
+<p>Комментарии можно вставлять в любое место в строке, но помните, что Python
+проигнорирует всё, что идёт после '#'.<p>
```
Comment (pluhin) [inline avodolazov/01_html/task4.html]: Where is the end of the file? body and html did not close

File: avodolazov/01_html/task5.html [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+def DoTwo(num1,num2):
+print '%d + %d = %d ' % (num1,num2,num1+num2)
+print '%d * %d = %d ' % (num1,num2,num1*num2)
+print '%d - %d = %d ' % (num1,num2,num1-num2)
+print '\n'
+DoTwo(1,2)
+DoTwo(1,4)
+DoTwo(10,5)<code></pre>
+<p>Как видите, здесь написано меньше кода — 8 строк вместо 12. Если нам нужно будет что-
+либо поменять в функции, мы сможем сделать это, не создавая проблем для нашей программы. Чтобы вызвать функцию, нужно написать её имя и параметры.</p>
```
Comment (pluhin) [inline avodolazov/01_html/task5.html]: Where is the end of the file? body and html did not close

File: avodolazov/01_html/task6.html [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+выходной: <u>суббота</u>, <u>воскресенье</u></p>
+<hr/>
+<h2>Вот такой текст</h2>
+<h1><sup>Э</sup><sub>т</sub><sup>о</sup><sub>п</sub><sup>р</sup><sub>о</sub><sup>с</sup><sub>т</sub><sup>о</sup><sub>в</sub><sup>е</sup><sub>с</sub><sup>ё</sup><sub>л</sub><sup>ы</sup><sub>й</sub><sup>т</sup><sub>е</sub><sup>к</sup><sub>с</sub><sup>т</sup></h1>
+<hr/>
+<h2>Акроним и курсив</h2>
+<p>&Tau;&Phi;: &Upsilon;(<i>x</i>)=e<sup><i>-i&pi;x</i></sup>+log<sub><i>x</i></sub>4</p>
+<hr/>
+<h2>Тег small внтури абзаца</h2>
+<p align="right"><small>Создано при помощи блокнота&copy;</small></p>
```
Comment (pluhin) [inline avodolazov/01_html/task6.html]: Where is the end of the file? body and html did not close

File: avodolazov/01_html/task3.html (hunk 5 above)
Comment (arester1980) [reply avodolazov/01_html/task3.html]: I close it on the next line. That's wrong?

File: avodolazov/02_site.by/about.html [hunk 9]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+  <HTML>
+  <HEAD>
+  <META charset="utf-8">
+  </HEAD>
+  <BODY>
+  <h2>Горизонтальное меню</h2>
+  <table>
+  <td><a href=index.html>Главная |</a></td>
```
Comment (pluhin) [inline avodolazov/02_site.by/about.html]: It is a good practices  If you put value of href in quotes

**Pull request merged**